src/loaders/lottie/tvgLottieBuilder.h

```diff
@@ -55,6 +62,7 @@ struct RenderContext
     LottieObject** begin = nullptr; //iteration entry point
     Array<RenderRepeater> repeaters;
     Matrix* transform = nullptr;
+    RenderOffsetPath* offsetPath = nullptr;
```

Comment:
I guess, in the end, it might also end up in the array like `RenderRepeater`... and all modifiers could be unified into a single array with `Array<LottieModifier>`...

We can ignore this for now.